src/main/java/org/demo/lesson2/tasks/inheritance/App.java

```diff
@@ -0,0 +1,22 @@
+package org.demo.lesson2.tasks.inheritance;
+
+/*
+ Shortly to say, inheritance is the OOP ability that allows Java classes to be derived from other classes.
+ */
+public class App {
+
+    public static void main(String[] args) {
+
+        VolleyPlayer PaolaEgonu = new VolleyPlayer("Italy", 186, "opposite");
```

Comment:
better to start variable name from lower case, like 

VolleyPlayer paolaEgonu
VolleyPlayer feGaray

etc.